src/Microsoft.AspNet.Razor/Generator/GeneratedTagHelperContext.cs

```diff
@@ -154,16 +100,21 @@ public GeneratedTagHelperContext()
         public string ExecutionContextTypeName { get; set; }
 
         /// <summary>
-        /// The name of the property used to encode HTML.
-        /// </summary>
-        public string HtmlEncoderPropertyName { get; set; }
-
-        /// <summary>
         /// The name of the type containing tag helper content.
         /// </summary>
         /// <remarks>
         /// Contains the data returned by EndTagHelperWriteScope().
         /// </remarks>
         public string TagHelperContentTypeName { get; set; }
+
+        /// <summary>
+        /// The Write method name for tag helpers.
+        /// </summary>
+        public string WriteTagHelperMethodName { get; set; }
+
+        /// <summary>
+        /// The WriteTo method name for tag helpers.
```

Comment:
rewrite these comments to be consistent w/ the rest of those in the class e.g. `The name of the method used to write <see cref="ExecutionContextTypeName"/> content.`